packages/dcos-ui-update-client/src/DCOSUIUpdateClient.ts

```diff
@@ -0,0 +1,57 @@
+import { Observable } from "rxjs";
+import { map } from "rxjs/operators";
+import { request, RequestResponse } from "@dcos/http-service";
+
+import Config from "#SRC/js/config/Config";
+import { getErrorMessage } from "./utils";
+
+import { UIVersionResponse } from "../";
+
+export const DCOSUIUpdateClient = {
+  fetchVersion: (): Observable<RequestResponse<UIVersionResponse>> => {
+    const resp$ = request(
+      `${Config.rootUrl || "/"}dcos-ui-update-service/api/v1/version/`
```

Comment:
I'm sorry having overlooked this. Let's try to avoid importing `#SRC` in packages. We shall instead make it an optional argument. Please note that in my `config.ts` I have `rootUrl: ""` which will result in `dcos-ui-update-service/api/v1/version/` without the leading slash.

Reply:
Is `Config.rootUrl` still used and useful? I added it after seeing it in the current [CosmosPackagesActions](https://github.com/dcos/dcos-ui/blob/master/src/js/events/CosmosPackagesActions.js#L62) because it seemed like it would be good to be consistent.

Reply:
We could keep the prefix just making it an argument instead of environment related global

Reply:
In which cases do we actually need the `rootUrl`? Does it even make sense to use it?

Reply:
@Poltergeist you are longer on the team :bad_poker_face: we could do git archeology in `mesosphere/dcos-ui`